spec/system/donation_system_spec.rb

```diff
@@ -178,7 +178,7 @@
           select Item.alphabetized.first.name, from: "donation_line_items_attributes_0_item_id"
           fill_in "donation_line_items_attributes_0_quantity", with: "5"
           page.find(:css, "#__add_line_item").click
-          select_id = page.find(:xpath, '//*[@id="donation_line_items"]/section[2]//select')[:id]
+          select_id = page.find(:xpath, '//*[@id="donation_line_items_container"]/div[2]/section/div/span[4]/div/select')[:id]
```

Comment:
ha. We should probably find a better way to do this, eh? :D

Reply:
Yeah :/ I'm not too familiar with this type of testing but I agree... Not a very intuitive way to check our views. I can try and figure out another way to check this using selenium but there's a lot of tests done this way throughout the specs. Maybe we should make an issue for this or note it in #1024? 

Let me know if this spec blocks the PR and I'll try and get it fixed soon!